automation/nightly/test-nightly-build.sh

```diff
@@ -67,9 +87,11 @@ make build-push-multi-arch-operator-image build-push-multi-arch-webhook-image bu
 
 # Update image digests
 sed -i "s#quay.io/kubevirt/virt-#${kv_image/-*/-}#" deploy/images.csv
+sed -i "s#quay.io/kubevirt/cluster-network-addons-operator#${cnao_image}#" deploy/images.csv
+
 sed -i "s#^KUBEVIRT_VERSION=.*#KUBEVIRT_VERSION=\"${kv_tag}\"#" hack/config
 sed -i "s#^CDI_VERSION=.*#CDI_VERSION=\"${cdi_tag}\"#" hack/config
-(cd ./tools/digester && go build .)
```

Comment:
can you explain what this line does and why is this removed?

Reply:
yes: it compiles a tool that finds the image digests from image tags. but the it already compiled within the `automation/digester/update_images.sh` script, so the line is actually redundant.